test/integration/goldens/showcase/samples/snippets/generated/main/java/com/google/showcase/v1beta1/compliance/create/SyncCreateSetCredentialsProvider.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.showcase.v1beta1.samples;
+
+// [START localhost7469_v1beta1_generated_Compliance_Create_SetCredentialsProvider_sync]
+import com.google.api.gax.core.FixedCredentialsProvider;
```

Comment:
We don't need these auto generated samples, they shouldn't be generated anymore without `include_samples=True`, the whole `samples` folder can be deleted.

Reply:
Actually they are here because how `include_samples` are implemented, it's implemented in `java_gapic_assembly_gradle_pkg` not `java_gapic_library`, so the samples are always generated but not included in the package. We may want to change this behavior though in the future.